SalesforceCommerceCloud/pwa-kit
# Fix pseudo locale

# Description

Use the "real" pseudo locale provided by react intl `en-XB`.

The locale `en-XB` is not documented yet but is provided by react intl along with the rest of the pseudo locales:
https://formatjs.io/docs/tooling/cli/#--pseudo-locale-pseudolocale

Use the default locale for the Commerce API request when using the pseudo locale `en-XB`.


# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Use the react intl pseudo locale `en-XB` instead of `pseudo`.
- Add guards for undefined values.
- Use the default locale for all the Commerce API requests when running the app using `npm run start:pseudolocale`.

# How to Test-Drive This PR

- Run the app using the pseudo locale:
```
npm run start:pseudolocale
```
- Verify that a pseudo locale value is a display for all the static string data.

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
